demisto_sdk/commands/content_graph/strict_objects/pre_process_rule.py

```diff
@@ -0,0 +1,57 @@
+from typing import Any, List, Optional
+
+from pydantic import Field
+
+from demisto_sdk.commands.content_graph.strict_objects.common import (
+    DESCRIPTION_DYNAMIC_MODEL,
+    NAME_DYNAMIC_MODEL,
+    SUFFIXED_ID_DYNAMIC_MODEL,
+    BaseStrictModel,
+    create_model,
+)
+
+
+class Period(BaseStrictModel):
+    by: Optional[str] = None
+    from_value: Optional[int] = Field(None, alias="fromValue")
+
+
+class _StrictPreProcessRule(BaseStrictModel):
+    action: str
+    enabled: bool
+    existing_events_filters: List[Any] = Field(
+        default_factory=list, alias="existingEventsFilters"
+    )
+    from_version: str = Field(alias="fromVersion")
+    id_: str = Field(alias="id")
+    index: int
+    item_version: str = Field(alias="itemVersion")
+    link_to: str = Field(alias="linkTo")
+    locked: bool
+    name: str
+    description: Optional[str] = None
+    new_event_filters: List[Any] = Field(default_factory=list, alias="newEventFilters")
```

Comment:
either model the values we have, or at least change to  
```suggestion
    new_event_filters: List[Dict[str, Union[str, Dict[str, Any]]]] = Field(default_factory=list, alias="newEventFilters")
```